libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/ApplyTx.hs

```diff
@@ -188,21 +186,20 @@ emptyCollInfo = CollInfo (Coin 0) (Coin 0) Set.empty Map.empty
 -- | Collect information about how to process Collateral, in a second phase failure.
 collInfo ::
   (Reflect era, HasCallStack) =>
-  Proof era ->
   Int ->
   TxIx ->
   Model era ->
   CollInfo era ->
   TxBodyField era ->
   CollInfo era
-collInfo proof count firstTxIx model info field = case field of
+collInfo count firstTxIx model info field = case field of
```

Comment:
Future proofing, possibly, but that is likely not going to be as much of a concern going forward with this API, because a lot of things we'll be able to do era agnostic without relying on `"proof"` concept